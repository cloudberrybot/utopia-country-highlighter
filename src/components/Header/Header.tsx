import React from 'react';

function Header() {
  return (
    <div className="flex items-center justify-center mt-6 mb-10 md:mt-[65px]">
      <h1 className="text-lg md:text-2xl lg:text-3xl max-w-2xl p-2 md:text-center text-black font-light">Select the region and click on the countries you want to highlight.</h1>
    </div>
  )
}

export default Header;