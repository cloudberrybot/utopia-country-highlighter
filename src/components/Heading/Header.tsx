interface IHeaderProps {
  content: string
}

function Header({ content }: IHeaderProps) {
  return (
    <div className="mt-8 ml-2 mb-2">
      <h1 className="text-lg text-gray-700 font-semibold">{content}</h1>
      <hr className="border-b border-gray-200 border-[1px] mt-1" />
    </div>
  )
}

export default Header;