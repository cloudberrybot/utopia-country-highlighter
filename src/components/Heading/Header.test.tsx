import Header from './Header'
import { render } from '@testing-library/react'

test('renders europe react link', () => {
  const { getByText } = render(
    <Header content="Header" />
  );

  expect(getByText(/Header/i)).toBeInTheDocument()
});
