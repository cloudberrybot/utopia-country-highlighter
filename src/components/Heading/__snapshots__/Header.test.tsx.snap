// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<DocumentFragment>
  <div
    class="mt-8 ml-2 mb-2"
  >
    <h1
      class="text-lg text-gray-700 font-semibold"
    >
      Header
    </h1>
    <hr
      class="border-b border-gray-200 border-[1px] mt-1"
    />
  </div>
</DocumentFragment>
`;
